packages/web-components/src/utils/date-utils.ts

```diff
@@ -16,18 +16,18 @@ const maxMonths = 12;
 const minMonths = 1;
 
 export const months = [
-  { label: 'January', value: 1 },
-  { label: 'February', value: 2 },
-  { label: 'March', value: 3 },
-  { label: 'April', value: 4 },
-  { label: 'May', value: 5 },
-  { label: 'June', value: 6 },
-  { label: 'July', value: 7 },
-  { label: 'August', value: 8 },
-  { label: 'September', value: 9 },
-  { label: 'October', value: 10 },
-  { label: 'November', value: 11 },
-  { label: 'December', value: 12 },
+  { label: () => i18next.t('january'), value: 1, key: 1 },
```

Comment:
I think the key value should not all be the same. Technically they should probably be unique. I think it'll be fine if they were just updated to be 1, 2, 3, etc. but ideally it would be a randomly generated value.

Reply:
Too much copy and paste.... Good catch